src/RecurringTimeInterface.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace RebelCode\Time;
+
+use Traversable;
+
+/**
+ * Represents a set of times, recurring at regular intervals, over a specific time period.
+ *
+ * @since [*next-version*]
+ */
+interface RecurringTimeInterface extends
```

Comment:
During a call, it was decided that this is to be removed. 2 main reasons:
- Sort of redundant, since you can just have a traversable with dates, without the need of interface, and without the rest of API.
- There isn't really a way to make sure that the dates in the list follow the period and interval restrictions.

Instead, somewhere else in the software, you could have a `createRecurringDates($period, $interval): Traversable`, which would *create and return* the recurring list of dates.